website/addons/figshare/tests/test_views.py

```diff
@@ -56,7 +58,21 @@ def setUp(self):
 
         self.figshare = create_mock_figshare('test')
 
+    def configure_responses(self):
```

Comment:
We are trying to remove usage of responses in favor of HTTPretty: https://github.com/CenterForOpenScience/osf.io/issues/2227 . Can you change these to use HTTPretty?